### ISSUE TITLE
Pretrained model release

### ISSUE THREAD

**Issue by Arjunsankarlal** [pytorch/fairseq#1266]
As I can see the table which shows the accuracy of base, large and mnli finetuned models performance of GLUE tasks, any plans to release the pretrained model of those models, since they would have been trained to evaluate :)

I tried finetuning robert-base for QQP task with transformers library, but I was able to acheive only,
acc = 0.6329982984448578
acc_and_f1 = 0.3164991492224289
f1 = 0.0

Also I have opened an [issue](https://github.com/huggingface/transformers/issues/1519) is their repo regarding this, and yet to get a response. Any leads would be helpful :)

**Comment by ngoyal2707** [comment]
@Arjunsankarlal There are no plans to release model for those downstream tasks. You can follow fairseq instructions to finetune the model on `QQP` tasks. `0.63` seems to low, did you make sure that you loaded the pretrained weights correctly?

**Comment by Arjunsankarlal** [comment]
Thanks for the prompt response @ngoyal2707. Yes, I loaded the pretrained model correctly as mentioned in their repo. Can you please direct me to the instructions where I can find the steps to finetune roberta model for downstream tasks ? I could only find preprocessing files. And commands [here](https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.glue.md) refers to which `train.py` file ?
